### DISCUSSION TITLE
Suggestions for GitHub Desktop commit search

### DISCUSSION THREAD

**Discussion by cwxr0423** [community/community#62634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello
    Sometimes I want to do a fuzzy search for commits in a branch based on the commit summary or description, but there doesn't seem to be a feature for that now, is it possible to add one? Thanks!

**Comment by DSDmark** [comment]
Hi @cwxr0423,

You can find by using syntax like this in search bar: `repo:<username>/<repo> is:commit "search term" in:branch <branch name>` like this:

![image](https://github.com/community/community/assets/85744816/0085149d-4810-4262-8cb9-334f761e9b4b)

And please check out this for [**more info about search syntax**](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax)
